src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Also I think if we need the rescue version of
freedos in a system-wide location for D, this
is not necessarily to handle with your scripts.
I think your scripts should concentrate on installing
any dos to C, and the rescue stuff can be handled
by the existing install.c code.
So it would really be nice to keep the use-case for
your scripts narrow.